bnd-maven-plugin/.gitignore

```diff
@@ -1 +1,4 @@
+/.classpath
+/.project
+/.settings/
```

Comment:
So in order to work on the "bnd-maven-plugin" project I imported into my eclipse using m2e plugin, which fires off the project configurators and generates those files.  So some developers will ignore these files since you can get them back whenever you "import" the project using m2e.

some people want these files checked in.  Both ways work I guess, what is bndtools pref and I can follow that way.